owned-manifests/machine-api-controllers.yaml

```diff
@@ -84,3 +84,15 @@ spec:
             cpu: 10m
             memory: 20Mi
       {{end}}
+      {{if .Controllers.ProviderRequestEnabled }}
```

Comment:
I don't see an end for this if statement in the template.

The template can test a string {{ if .Controllers.ProviderRequest }} so we may not need the extra boolean flag.